### PR TITLE
Feat/#11 dashboard cards

### PR DESCRIPTION
**Pull request by ciaranschutte**
Basic dashboard page with cards

myapplications

### CODE REVIEW

**Comment by ciaranschutte** [comment]
@caravinci adding rest of card content in different ticket

File: components/pages/Applications/Dashboard/Header/AccessBox.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      css={css`
+        align-self: flex-start;
+        border: 1px solid ${theme.colors.grey_2};
+        border-radius: 8px;
+        display: flex;
+        align-items: center;
+        flex: 0 0 379px;
+      `}
+    >
+      <IconWithText hasAccess={hasAccess}>
```
Comment (caravinci) [inline components/pages/Applications/Dashboard/Header/AccessBox.tsx:46]: same with this `hasAccess` props, is it for a future implementation?

**Review by caravinci — APPROVED** [review]
Looks pretty good overall.
Just a couple questions (with no need for changes, just asking for purpose/intent)

File: components/pages/Applications/Dashboard/Header/AccessBox.tsx (hunk 1 above)
Comment (ciaranschutte) [reply components/pages/Applications/Dashboard/Header/AccessBox.tsx:46]: yep for future implementation

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  grid-auto-columns: 1fr;
+  grid-auto-flow: column;
+`;
+
+const Applications = ({ inProgressApplications = true }: ApplicationsProps) => {
+  const theme: UikitTheme = useTheme();
+
+  return (
+    <ContentBody>
+      <TwoColGrid inProgress={true}>
```
[the remark this answers is not in the document]
Comment (ciaranschutte) [reply components/pages/Applications/Dashboard/Applications/index.tsx:24]: this is not actually needed anymore seen as the grid config should align the cards
I was originally passing this in to do a custom value in the styling

**Pull request merged**